app/views/users/_account.html.erb

```diff
@@ -14,6 +14,40 @@
   </div>
 <% end %>
 
+<h2>Access Tokens</h2>
+<p>You can generate personal access tokens to use for authentication with the DEV API.</p>
+<h3>Generate an access token</h3>
+<h4>Token description</h4>
+<%= form_tag users_api_secrets_path, method: :post do %>
+  <%= fields_for :api_secret do |api_secret| %>
+    <div class="field">
+      <%= api_secret.text_field(:description, placeholder: "What's this token for?", required: true) %>
+    </div>
+  <% end %>
+  <button class="big-action cta" id="new__api__secret__btn" type="submit">
+    Generate Token
+  </button>
+<% end %>
+
+<h3>Active access tokens</h3>
+<% if @user.api_secrets.empty? %>
+  <p>None yet!</p>
+<% end %>
+<% @user.api_secrets.order(created_at: :desc).each do |api_secret| %>
```

Comment:
Nit. What do you think of moving this query to the controller?

This way, the view doesn't have to know anything about the ordering logic.